### DISCUSSION TITLE
CICD?

### DISCUSSION THREAD

**Discussion by BetimBeja** [Power-Platform-Patterns-and-Practices/Dataverse.Sdk.Extensions#11] [Ideas]
Hello @MscrmTools ,
any plans to implement a CICD pipeline for this project? Personally, I have more experience with Azure Pipelines, and could suggest a valid pipeline for CICD, otherwise, I want to experiment a little with GitHub Actions.

Other preferences?

**Comment by MscrmTools** [comment]
I guess that would be nice to use Github action to have everything in the same place. But like you, I'm more at ease with Azure DevOps than Github

**Reply by BetimBeja** [reply]
I managed to do this for one of my repositories https://github.com/albanian-xrm/Xrm-Entity-Serializer/pull/6#issuecomment-835818859
